src/sentry/feedback/usecases/create_feedback.py

```diff
@@ -360,22 +367,22 @@ def shim_to_feedback(
                 feedback_event["contexts"]["trace"] = event.data["contexts"]["trace"]
 
             feedback_event["timestamp"] = event.datetime.timestamp()
-            feedback_event["level"] = event.data["level"]
             feedback_event["platform"] = event.platform
             feedback_event["level"] = event.data["level"]
             feedback_event["environment"] = event.get_environment().name
-            feedback_event["tags"] = [list(item) for item in event.tags]
 
-        else:
+            tags = {}
+            for [k, v] in event.tags:
+                tags[k] = v
+            feedback_event["tags"] = tags
+
             metrics.incr(
                 "feedback.user_report.missing_event",
                 sample_rate=1.0,
                 tags={"referrer": source.value},
             )
 
             feedback_event["timestamp"] = datetime.utcnow().timestamp()
-            feedback_event["platform"] = "other"
-            feedback_event["level"] = report.get("level", "info")
```

Comment:
`level` was being set 3 times

Reply:
lets do the level and platform changes in a different PR

Reply:
the level change i have to make now bc it broke a test, is it worth separating out the platform one if it's just one line?

Reply:
Oh ok, no they look fine to keep then